dipdup-io/stone-packaging
# Integrate Vocs framework and add documentation setup instructions

This PR addresses issue #21 by integrating the Vocs framework to generate a static site from our markdown documentation files.

Changes made:
1. Installed and configured Vocs for the project
2. Organized existing markdown files in the `docs/pages` directory
3. Created a `vocs.config.ts` file to configure the documentation site
4. Updated the README.md with detailed instructions on:
   - Prerequisites for running the documentation site
   - Installation steps
   - How to run the development server
   - How to build the static site
   - Project structure overview
   - Process for adding new pages to the documentation

To test these changes:
1. Clone the repository
2. Follow the installation instructions in the README
3. Run the development server and verify that the documentation site loads correctly
4. Build the static site and check that it generates without errors



Closes #21
